packages/@sfx/sayt-plugin/src/sayt-plugin.ts

```diff
@@ -0,0 +1,34 @@
+import { Plugin, PluginRegistry, PluginMetadata } from '@sfx/core';
+import { Sayt, SaytConfig } from 'sayt';
+/**
+ * This plugin is responsible for exposing an instance of sayt
+ * to Core.
+ */
+ export default class SaytPlugin implements Plugin {
+  get metadata(): PluginMetadata {
+    return {
+      name: 'sayt',
+      depends: [],
+    };
+  }
+  /**
+   * The sayt property is the value that the Sayt Plugin
+   * exposes to the Core entity.
+   */
+  sayt: Sayt;
+  /**
+   * The sayt plugin constructor function can take in optional options. It also instantiates
+   * an instance of sayt, with any given options.
```

Comment:
I would say something like "This plugin's constructor. It instantiates an instance of the sayt plugin and attaches it to this plugin's `sayt` property."